lnrpc/rpc.proto

```diff
@@ -2516,6 +2527,32 @@ message ChannelGraph {
     repeated ChannelEdge edges = 2;
 }
 
+enum NodeMetricType {
+    BETWEENNESS_CENTRALITY = 0;
+}
+
+message NodeMetricsRequest {
+  /// The requesteded node metrics.
+  repeated NodeMetricType types = 1;
+}
+
+message NodeMetricsResponse {
+    /// Node metrics by node pubkey.
+    map<string, NodeMetrics> metrics = 1;
```

Comment:
how will `NodeMetrics` work for other metrics? Since it has centrality as explicit fields.

Reply:
Changed as we discussed offline. PTAL

Reply:
I'm okay doing it this way.